api/base.go

```diff
@@ -255,6 +256,19 @@ func (h *BaseHandler) modBody(
 	return
 }
 
+//
+// CurrentUsername gets username from Keycloak auth token
+func (h *BaseHandler) CurrentUsername(ctx *gin.Context) (username string) {
```

Comment:
Just a preference.  `CurrentUser()`.  I generally stay away from _username_ in all cases because it _could_ be confused wth the User's name like "John Doe".